src/wallet/wallet2.cpp

```diff
@@ -8891,29 +9442,31 @@ std::vector<wallet2::pending_tx> wallet2::create_transactions_2(std::vector<cryp
   // early out if we know we can't make it anyway
   // we could also check for being within FEE_PER_KB, but if the fee calculation
   // ever changes, this might be missed, so let this go through
-  const uint64_t min_fee = (fee_multiplier * base_fee * estimate_tx_size(use_rct, 1, fake_outs_count, 2, extra.size(), bulletproof));
-  uint64_t balance_subtotal = 0;
-  uint64_t unlocked_balance_subtotal = 0;
-  for (uint32_t index_minor : subaddr_indices)
+  const uint64_t min_outputs = tx_params.tx_type == cryptonote::txtype::key_image_unlock ? 1 : 2;
   {
-    balance_subtotal += balance_per_subaddr[index_minor];
-    unlocked_balance_subtotal += unlocked_balance_per_subaddr[index_minor].first;
+    uint64_t min_fee = (base_fee * estimate_tx_size(1, fake_outs_count, min_outputs, extra.size()) + base_fee * min_outputs) * fee_algorithm / 100;
```

Comment:
'fee_algorithm' was not declared in this scope; did you mean 'OBJ_algorithm'?
On windows compiler